03-interactive-site/00-html-and-css/02-css/README.md

```diff
@@ -0,0 +1,230 @@
+# Introdução a CSS
+
+- Tipo: `leitura`
+- Formato: `ritmo próprio`
```

Comment:
Let's use `guiado` or `individual` please. At the end, it will be converted to `guided` or `self-paced` independently from the locale.